lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss

```diff
@@ -1,14 +1,72 @@
 @charset "UTF-8";
 
+
+// For all templates.
+.views-view-grid {
```

Comment:
This selector is a little generic and probably should be locked down. We can add a specific css class to the wrapper of the view if you need it.

Reply:
Ok cool; on it.

Reply:
I fixed this and added a new class so there is a config yml change.